src/autopas/containers/CellBlock3D.h

```diff
@@ -278,9 +273,19 @@ class CellBlock3D : public CellBorderAndFlagManager {
   index_t getLastOwnedCellIndex() const;
 
  private:
-  [[nodiscard]] std::array<index_t, 3> index3D(index_t index1d) const;
+  /**
+   * Transform a 1D index to a 3D index in the context of this cell block.
+   * @param index1D
+   * @return 3D cell index.
+   */
+  [[nodiscard]] std::array<index_t, 3> oneToThreeD(index_t index1D) const;
 
-  [[nodiscard]] index_t index1D(const std::array<index_t, 3> &index3d) const;
+  /**
+   * Transform a 3D index to a 1D index in the context of this cell block.
+   * @param index3D
+   * @return 1D cell index.
+   */
+  [[nodiscard]] index_t threeToOneD(const std::array<index_t, 3> &index3D) const;
```

Comment:
What is meant by "context"

Reply:
The context is the current `CellBlock3D` object